assets/src/store.js

```diff
@@ -1,7 +1,7 @@
-import Vue from 'vue'
-import Vuex from 'vuex'
+import Vue from 'vue';
+import Vuex from 'vuex';
 
-Vue.use(Vuex)
+Vue.use(Vuex);
```

Comment:
Are you actually using Vuex? Is this file necessary?

Reply:
You're right, removed in 968b3cf!